code/datums/components/chameleon/spritesheet.dm

```diff
@@ -0,0 +1,41 @@
+/datum/asset/spritesheet/chameleon
+	name = "chameleon"
+
+/datum/asset/spritesheet/chameleon/create_spritesheets()
+	var/list/disguises = list()
+	// First, get everything we need to make icons of.
+	for(var/chameleon_path in subtypesof(/datum/component/chameleon))
+		var/datum/component/chameleon/chameleon = chameleon_path
```

Comment:
```suggestion
		var/datum/component/chameleon/chameleon = New chameleon_path()
		// snip
		list_chameleon_disguises(..., chameleon.disguise_whitelist, ...)
		del(chameleon)
```
You can't use `initial()` to a var/list. You need to create a type to get a list, then del it later.